helpers/gestures.js

```diff
@@ -0,0 +1,109 @@
+const maxTime = 2000;
+const minDistance = 30;
+
+export function registerSwipe(node) {
+	node.addEventListener('touchstart', handleTouchStart);
```

Comment:
Does this ever need to be unregistered?

Reply:
Good question.  Currently, it would only get removed when the element is disposed, similar to how we wire up events to elements internal to a Lit component.  The original impl provided an unregister method (that was never used) that would be useful if a consumer wants to stop listening for touch events but they still want to keep the element.  Do you think I should provide that?

Reply:
Would it even get removed though on disposal? I guess unlike events that get wired up to the document, touch events on the element aren't going to continue to fire long after the element's been removed from the DOM.

Reply:
Yeah, it's just the ones that get wired-up outside that scope that are of concern.  

That's the approach being used in Lit components as well - the topic comes up every once a while in the Polymer Slack, with the general recommendation being to simply wire-up in `firstUpdated` for the `host`, although I think Justin mentioned it can be delayed even further by wiring up in `constructor() { setTimeout(() => { this.addEventListener... }, 0) }`, and only using `connectedCallback`/`disconnectedCallback` when wiring up to things outside its scope.  Long story short, they also assume handlers are cleaned up when the node is cleaned up.

That all being said, the unregister _could_ be useful to stop listening for touch... in effective disabling swipe for some reason.  However the consumer can also accomplish that by removing their `d2l-swipe` handler.  🤷‍♂